DragAndDrop/sample/src/main/java/com/microsoft/device/dualscreen/draganddrop/ui/view/DragPane.kt

```diff
@@ -0,0 +1,113 @@
+import androidx.compose.ui.graphics.drawscope.Stroke
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import com.microsoft.device.dualscreen.draganddrop.DragData
+import com.microsoft.device.dualscreen.draganddrop.DragTarget
+import com.microsoft.device.dualscreen.draganddrop.MimeType
+import com.microsoft.device.dualscreen.draganddrop.R
+import com.microsoft.device.dualscreen.draganddrop.ui.theme.Purple500
+
+@Composable
+fun DragPane(modifier: Modifier = Modifier) {
+    Box(
+        modifier = modifier.fillMaxSize(),
+        contentAlignment = Alignment.Center
+    ) {
+        Column(
+            horizontalAlignment = Alignment.CenterHorizontally,
+            verticalArrangement = Arrangement.SpaceBetween
+        ) {
+            DragTextBox(Modifier.weight(.3f))
+            Spacer(modifier = Modifier.height(20.dp))
+            DragImageBox(Modifier.weight(.7f))
+        }
+    }
+}
+
+@Composable
+fun DragTextBox(modifier: Modifier) {
+    val dragText = stringResource(R.string.drag_text)
+    val dragData = DragData(type = MimeType.TEXT_PLAIN, data = dragText)
+
+    DragTarget(dragData = dragData) {
+        Box(
+            modifier = modifier.padding(top = 20.dp),
+            contentAlignment = Alignment.Center
+        ) {
+            Canvas(
+                modifier = Modifier.size(450.dp, 60.dp)
+            ) {
+                drawRoundRect(
+                    color = Purple500,
+                    cornerRadius = CornerRadius(20f),
+                    style = Stroke(
+                        width = 5f,
+                        pathEffect = PathEffect.dashPathEffect(
+                            intervals = floatArrayOf(20f, 10f),
+                            phase = 10f
+                        )
+                    )
+                )
+            }
+            Text(
+                text = dragText,
+                textAlign = TextAlign.Center,
+                style = MaterialTheme.typography.h5
+            )
+        }
+    }
+}
+
+@Composable
+fun DragImageBox(modifier: Modifier) {
+    val dragImage = painterResource(id = R.drawable.drag_and_drop_image)
+    val dragData = DragData(type = MimeType.IMAGE_JPEG, data = dragImage)
+
+    Box(
+        contentAlignment = Alignment.Center,
+        modifier = modifier.size(500.dp, 150.dp)
+    ) {
+        Row(verticalAlignment = Alignment.CenterVertically) {
+            Text(
+                text = stringResource(id = R.string.drag_action),
+                textAlign = TextAlign.Center,
+                style = MaterialTheme.typography.h5,
+                modifier = Modifier.weight(1f)
+            )
+            DragTarget(dragData = dragData) {
+                Image(
+                    painter = dragImage,
+                    contentDescription = null,
+                    modifier = Modifier.weight(1f)
```

Comment:
small detail, but I think it would look nice if the image had matching rounded corners - what do you think?
```suggestion
                    modifier = Modifier.weight(1f).clip(MaterialTheme.shapes.large)
```